src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/ILLink/ILLinkTrim.xml

```diff
@@ -2,4 +2,12 @@
     <assembly fullname="System.Private.Runtime.InteropServices.JavaScript">
         <type fullname="System.Runtime.InteropServices.JavaScript.Runtime" />
     </assembly>
+    <!--
+    Preserve System.Delegate.DynamicInvoke which is called from javascript
+    -->
+    <assembly fullname="mscorlib">
+      <type fullname="System.Delegate">
+        <method signature="System.Object DynamicInvoke(System.Object[])" />
```

Comment:
Is it necessary to call via this slow path? Cannot we simply call Invoke directly or use even faster path?

@lewing

Reply:
This is the only way to call this from JavaScript.  Look up the assembly and get a reference to the object of DynamicInvoke and then call the driver.

Reply:
Will talk with Larry tonight

Reply:
Took more of a look at this and we would need to add generics support to the bindings interface.